compiler/rustc_feature/src/active.rs

```diff
@@ -487,6 +487,8 @@ declare_features! (
     (incomplete, repr128, "1.16.0", Some(56071), None),
     /// Allows `repr(simd)` and importing the various simd intrinsics.
     (active, repr_simd, "1.4.0", Some(27731), None),
+    /// Allows return-position `impl Trait` in traits.
+    (incomplete, return_position_impl_trait_in_trait, "CURRENT_RUSTC_VERSION", Some(91611), None),
```

Comment:
shouldn't we need to change `CURRENT_RUSTC_VERSION` to the right version number?

Reply:
no it's a placeholder that gets replaced when beta is promoted

Reply:
This is awesome then :smile: